Rover/Rover.h

```diff
@@ -62,9 +63,6 @@
 #include "GCS_Rover.h"
 #include "AP_Rally.h"
 #include "RC_Channel.h"                  // RC Channel Library
-#if PRECISION_LANDING == ENABLED
-#include <AC_PrecLand/AC_PrecLand.h>
-#endif
```

Comment:
Need to include AC_Precland.h here if AC_PRECLAND_ENABLED (making it look like Copter)